autoload/buffet.vim

```diff
@@ -82,6 +82,11 @@ function! buffet#update()
     elseif s:last_current_buffer_id == -1 && len(s:buffer_ids) > 0
         let s:last_current_buffer_id = s:buffer_ids[0]
     endif
+
+    " Hide tabline if only one buffer and tab open
+    if !g:buffet_always_show_tabline && len(s:buffer_ids) == 1 && tabpagenr("$") == 1
+        set showtabline=0
+    endif
```

Comment:
same here, you can drop the `tabpagenr("$") == 1` part.

Reply:
we want to show the tabline if there's one buffer, but two or more tabs open

Reply:
oh now I see, makes sense 👍